test/inductor/test_torchinductor.py

```diff
@@ -3111,6 +3111,7 @@ def forward(self, l_input_: torch.Tensor):
         o2 = torch.compile(mod)(inp)
         self.assertEqual(o1, o2)
 
+    @patch.object(config.trace, "enabled", True)
```

Comment:
why this change?

Reply:
To add a test with "TORCH_COMPILE_DEBUG=1". It seems that we don't have any test coverage for "TORCH_COMPILE_DEBUG=1" for now.

Reply:
Should probably just make a separate sanity test.

Reply:
@Chillee Where do you suggest to put this sanity test?